compiler+runtime/test/jank/form/case/fail-duplicate-keys-return.jank

```diff
@@ -0,0 +1,8 @@
+; This will fail due to duplicate keys
+; test case in return position
+(=
+ (case 1
+   1 :one
+   1 :duplicate-one
+   :default)
+ :one)
```

Comment:
Every one of these test files is missing a newline at the end. Will you please configure your IDE to add that?